### ISSUE TITLE
Incorrect report export

### ISSUE THREAD

**Issue by abbyad** [medic/medic-webapp#998]
As shown in https://github.com/medic/medic-projects/issues/24, the export only works from the Configuration > Export menu, not from the top level Export.

Instead of getting ~50 records, it gets the following (header row only):

```
<?xml version="1.0" encoding="UTF-8"?>
<?mso-application progid="Excel.Sheet"?>
<Workbook xmlns="urn:schemas-microsoft-com:office:spreadsheet"
 xmlns:o="urn:schemas-microsoft-com:office:office"
 xmlns:x="urn:schemas-microsoft-com:office:excel"
 xmlns:ss="urn:schemas-microsoft-com:office:spreadsheet"
 xmlns:html="http://www.w3.org/TR/REC-html40">
<Worksheet ss:Name="Nh006"><Table><Row><Cell><Data ss:Type="String">Record UUID</Data></Cell><Cell><Data ss:Type="String">Patient ID</Data></Cell><Cell><Data ss:Type="String">Reported Date</Data></Cell><Cell><Data ss:Type="String">From</Data></Cell><Cell><Data ss:Type="String">Clinic Contact Name</Data></Cell><Cell><Data ss:Type="String">Clinic Name</Data></Cell><Cell><Data ss:Type="String">Health Center Contact Name</Data></Cell><Cell><Data ss:Type="String">Health Center Name</Data></Cell><Cell><Data ss:Type="String">District Hospital Name</Data></Cell></Row></Table></Worksheet></Workbook>
```

This instance is running v0.4.6.

**Comment by mandric** [comment]
It looks like an issue with constructing the query URL, the form code needs to be forced to uppercase in the query if the form code is lowercase in the form definition.  Example:

works:

fails

mandric in ~/dev/medic-webapp 
$ curl -u admin:123qwe 'http://127.0.0.1:5988/medic/_design/medic/_rewrite/export/forms/P?startkey=%5B%22R%22%2C9999999999999%5D&endkey=%5B%22R%22%2C0%5D&tz=180&format=csv&reduce=false'
"Record UUID","Patient ID","Reported Date","From","Nombre de Contacto en la Clínica","Nombre de la clínica","Nombre de Contacto en el Centro de Salud","Nombre del Centro de Salud","Nombre Hospital de Distrito","Weeks since LMP","Name"
"dba419d5a09f22712162ce366a013fe6","49827","19, Jun 2014, 08:42:00 -03:00","+2840915732","Janet","Sample Catchment Area 1","Mary","Sample Health Center","Sample District","11","Prudence"

Fails:
```$ curl -u admin:123qwe 'http://127.0.0.1:5988/medic/_design/medic/_rewrite/export/forms/p?startkey=%5B%22R%22%2C9999999999999%5D&endkey=%5B%22R%22%2C0%5D&tz=180&format=csv&reduce=false'

**Comment by mandric** [comment]
@garethbowen any thoughts about this patch?

**Comment by mandric** [comment]
Pushed to beta.

**Comment by abbyad** [comment]
@mandric was this included in 4.8?

**Comment by mandric** [comment]
Yep should be.  It's in the changes log and commits are tagged.
